.babelrc

```diff
@@ -1,9 +1,10 @@
 {
   "presets": [
+    "stage-1",
     "react",
     ["env", {
       "modules": false
     }]
   ],
-  "plugins": ["transform-class-properties"]
+  "plugins": ["transform-class-properties", "transform-async-to-generator"]
```

Comment:
I don't think you need `transform-async-to-generator` plugin. ES2017 includes async and await feature. Just specify 2 last  versions of browsers for `env` plugin